app/.env.symlink

```diff
@@ -1 +0,0 @@
-../.env
```

Comment:
This file shouldn't be deleted.

Reply:
Okay. I have added .env.symlink from ira main branch

Reply:
This file should be a symlink. Ideally, changes to this file shouldn't be even here in diff.